index.html

```diff
@@ -3197,11 +3208,10 @@ <h2 id="changes-from-first-draft">Changes from 24 November 2020 First Public Wor
 
     <section id="acknowledgements" class="appendix normative">
         <h1>Acknowledgments</h1>
-<!--
-        <p>Special thanks to X, Y, and Z
-           for their contributions to this document.
+        <p>Special thanks to Arne Broering, Soumya Kanti Datta, and Christian Bonnet
+           for their survey work on existing discovery mechannisms in IoT.
```

Comment:
@mmccool I just noticed that a small typo was introduced in this line ("mechannisms" instead of "mechanisms").